lib/app/features/feed/create_article/views/pages/create_article_modal/hooks/use_create_article.dart

```diff
@@ -44,12 +48,19 @@ CreateArticleState useCreateArticle() {
     [titleController],
   );
 
+  void onNext(WidgetRef ref) {
+    ref.read(createArticleProvider.notifier).updateImage(selectedImage.value);
+    ref.read(createArticleProvider.notifier).updateTitle(titleController.text);
+    ref.read(createArticleProvider.notifier).updateContent(textEditorController);
```

Comment:
why do we need to update it separately instead update all at ones?

Reply:
Can we decrease state calls? If an image and text come at the same time, we can use one update function. If data comes separately, then it's okay.